### PR TITLE
Updated readme - add maven project

### PR DESCRIPTION
**Pull request by jvolmutsl**
Added a little note to the readme on how to run a project in intelliJ when the whole repo is loaded.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
 # Design Patterns Study Group
 
 Contains notes and exercises done as part of the Bristol DP study group.
+
+## Add maven project
+In case you open the whole repository in the IntelliJ and it doesn't give you an opportunity to run the project:
```
Comment (fbromley) [inline README.md]: I don't think you need the before IntelliJ, but you could keep it in for dramatic effect!

File: README.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+1. Right click on a pom.xml of the particular project
```
Comment (fbromley) [inline README.md]: Right click on 'pom.xml' for the project you wish to run

**Review by fbromley — COMMENTED** [review]
couple of suggestions :)

File: README.md (hunk 1 above)
Comment (jvolmutsl) [reply README.md]: The 'the'? 😅

File: README.md (hunk 2 above)
Comment (jvolmutsl) [reply README.md]: Thank you, added :)

**Review by tillywoodfield — APPROVED** [review]

**Pull request merged**